.github/workflows/docker-build.yml

```diff
@@ -4,10 +4,11 @@ on:
   push:
```

Comment:
> I presume it isn't running the github action because there's a disconnect between the github action that's configured on the main branch and the one in the PR.

It isn't running because you have the trigger to only be push, tags, and workflow dispatch. You need to add `pull_request` for it to trigger here.

Reply:
See a test example here: https://github.com/jbowen93/celestia-node/pull/8 

Once the GH action configuration is on main pushes to other branches will also trigger a build during PRs.